themes/ddbasic/sass/module/_carousel.scss

```diff
@@ -461,3 +453,27 @@ $carousel-tab-color:                       $white;
     font-size: 80%;
   }
 }
+
+.content-wrapper {
```

Comment:
Ok, I agree. It's a "global" selector. No problem :)
Remove the empty selector in the beginning of the file and all is  OK 👍